src/it/scala/djinni/IntegrationTest.scala

```diff
@@ -64,35 +64,76 @@ class IntegrationTest extends FunSpec {
   }
 
   /**
-    * Executes the djinni generator with the given idl-file as input
+    * Generates the command line parameters to pass to the djinni generator.
+    *
+    * @param idl filename of the djinni-file (without file extension). The file must be located in the
+    *            `resources`-folder of the integration-tests (`src/it/resources`)
+    * @param baseOutputPath The root folder for the outputs to be generated.
+    * @param cpp Whether to generate C++ output. Default: true.
+    * @param java Whether to generate Java output. Default: true.
+    * @param objc Whether to generate Objective C output. Default: true.
+    * @param python Whether to generate Pyhton output. Default: true.
+    * @param cWrapper Whether to generate Pyhton output. Default: true.
+    * @param cppCLI Whether to generate Pyhton output. Default: true.
```

Comment:
```suggestion
    * @param python Whether to generate Python output. Default: true.
    * @param cWrapper Whether to generate Python output. Default: true.
    * @param cppCLI Whether to generate Python output. Default: true.
```

I spotted the typos! ;)